From 9f5afd99cce93e68996deb2b5fa7c32737d279fe Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Tue, 16 Jul 2024 17:54:29 +0800
Subject: [PATCH] cmake: remove build flags

Don't generate the build host related info and reemove the build flags
to improve reproducibility.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 config/cmake/libhdf5.settings.cmake.in | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/config/cmake/libhdf5.settings.cmake.in b/config/cmake/libhdf5.settings.cmake.in
index deb07ed..6f255c4 100644
--- a/config/cmake/libhdf5.settings.cmake.in
+++ b/config/cmake/libhdf5.settings.cmake.in
@@ -23,23 +23,23 @@ Linking Options:
 ----------------
                       Libraries: @BUILD_NAME_EXT@
   Statically Linked Executables: @BUILD_STATIC_EXECS@
-                        LDFLAGS: @CMAKE_SHARED_LINKER_FLAGS@
+                        LDFLAGS:
                      H5_LDFLAGS: @H5_LDFLAGS@
                      AM_LDFLAGS: @AM_LDFLAGS@
                 Extra libraries: @LINK_LIBS@
-                       Archiver: @CMAKE_AR@
-                       AR_FLAGS:
-                         Ranlib: @CMAKE_RANLIB@
+                       Archiver:
+                       AR_FLAGS:
+                         Ranlib:
 
 Languages:
 ----------
                               C: YES
-                     C Compiler: @CMAKE_C_COMPILER@ @CMAKE_C_COMPILER_VERSION@
+                     C Compiler:
                        CPPFLAGS: @CPPFLAGS@
                     H5_CPPFLAGS: @H5_CPPFLAGS@
                     AM_CPPFLAGS: @AM_CPPFLAGS@
-                         CFLAGS: @CMAKE_C_FLAGS@
-                      H5_CFLAGS: @HDF5_CMAKE_C_FLAGS@
+                         CFLAGS:
+                      H5_CFLAGS:
                       AM_CFLAGS: @AM_CFLAGS@
                Shared C Library: @H5_ENABLE_SHARED_LIB@
                Static C Library: @H5_ENABLE_STATIC_LIB@
@@ -51,11 +51,11 @@ Languages:
                AM Fortran Flags: @AM_FCFLAGS@
          Shared Fortran Library: @H5_ENABLE_SHARED_LIB@
          Static Fortran Library: @H5_ENABLE_STATIC_LIB@
-               Module Directory: @CMAKE_Fortran_MODULE_DIRECTORY@
+               Module Directory:
 
                             C++: @HDF5_BUILD_CPP_LIB@
                    C++ Compiler: @CMAKE_CXX_COMPILER@ @CMAKE_CXX_COMPILER_VERSION@
-                      C++ Flags: @CMAKE_CXX_FLAGS@
+                      C++ Flags:
                    H5 C++ Flags: @HDF5_CMAKE_CXX_FLAGS@
                    AM C++ Flags: @AM_CXXFLAGS@
              Shared C++ Library: @H5_ENABLE_SHARED_LIB@
-- 
2.25.1

